pkgs/by-name/ri/river-bsp-layout/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "river-bsp-layout";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "areif-dev";
+    repo = "river-bsp-layout";
+    rev = "v${version}";
+    hash = "sha256-LRVZPAS4V5PtrqyOkKUfrZuwLqPZbLoyjn2DPxCFE2o=";
+  };
+
+  cargoHash = "sha256-CtVyRwfIS8R48LUecKXoak+HHB5yNZ5RgguIWOhyFA8=";
+
+  meta = with lib; {
```

Comment:
Using `with lib` here and then `with lib.maintainers` below is considered bad practice because you are reaching outside the scope of the `with`. In this case, you could change the latter to just `with maintainers` to fix that issue but it's also the case that folks recommend against using `with` period these days so I would suggest instead just removing it from this line and using `lib.licenses` below. IMO you can continue to use `with lib.maintainers` because that is such a small localized scope.